controllers/update.go

```diff
@@ -513,9 +513,10 @@ func getComponentName(gitSource *appstudiov1alpha1.GitSource) string {
 // - End with an alphanumeric character
 // - Must not contain all numeric values
 func sanitizeComponentName(name string) string {
-	exclusive := regexp.MustCompile(`[^a-zA-Z0-9/-]`)
+	exclusive := regexp.MustCompile(`[^a-zA-Z0-9-]`)
```

Comment:
so does this regex say no `/` in component name?

Reply:
correct. `/` is not a valid character for  DNS1123 name. I might accidentally include that in the regex last time.